app/src/main/java/org/gdg/frisbee/android/widget/FeedbackFragment.java

```diff
@@ -71,8 +73,17 @@
     private static final String PROPERTY_APP_VERSION_CODE = "App Version Code";
 //    private static final String POWERED_BY_DOORBELL_TEXT = "Powered by <a href=\"https://doorbell.io\">Doorbell.io</a>";
 
-    @Bind(R.id.feedback_message_text) EditText mMessageField;
-    @Bind(R.id.feedback_email_text) AutoCompleteTextView mEmailField;
+    @Bind(R.id.feedback_message_text)
+    EditText mMessageField;
+
+    @Bind(R.id.feedback_email_text)
+    AutoCompleteTextView mEmailField;
+
+    @Bind(R.id.feedback_message_text_layout)
+    TextInputLayout mLayoutMessage;
```

Comment:
@friedger it's already defined in dialog feedback layout https://github.com/gdg-x/frisbee/blob/develop/app/src/main/res/layout/dialog_feedback.xml